### DISCUSSION TITLE
site.url access (from bridgetown.config.yml) in erb files

### DISCUSSION THREAD

**Discussion by btihen** [bridgetownrb/bridgetown#601] [Q&A]
in [bridgetown.config.yml](https://www.bridgetownrb.com/docs/variables) - I read that I can access site.url in liquid files, when I try to access `site` variables within an erb (in the head partial) I get `undefined method`.  

```
Exception raised: NoMethodError
[Bridgetown] undefined method `url' for #<Bridgetown::Site "title"=>"btihen", "tagline"=>"Dev Notes", "author"=>"Bill Tihen", "email"=>"your-email@example.com", "image"=>"/images/logo_mnt_shaded.svg", "description"=>"Mostly Dev article exploration and notes, so I don't forget & maybe help others", "twitter"=>{"username"=>"btihen", "card"=>"summary"}> 
```

As you can see, I do however, have access with `site` to all the same data as I do with metadata from `_data/metadata.config.yml` -  but not `bridgetown.config.yml`

This is not how I understand the docs - am I doing something wrong?  How do I access `bridgetown.config.yml` data in an erb file? Am I doing something incorrectly?

bridgetown 1.0.0
ruby 3.1.2
MocOS 12.4 (21F79) - M1

**Comment by jaredcwhite** [comment]
@btihen From Ruby files (ERB, etc.), you can use `site.config`. `site.config.url`, etc.

More details in the docs here: https://www.bridgetownrb.com/docs/template-engines/erb-and-beyond
